src/test/scala/nl.knaw.dans.easy.bag2deposit/AppSpec.scala

```diff
@@ -86,6 +77,13 @@ class AppSpec extends AnyFlatSpec with Matchers with AppConfigSupport with FileS
     val validBag = resourceBags / validUUID / "bag-revision-1"
     val movedBag = testDir / "ingest-dir" / validUUID / "bag-revision-1"
 
+    // DDM should have preserved its white space
```

Comment:
the merge lost at least one line:
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/pull/33/files#diff-9cb678bca90469930a7f58459c89d1ab3a2784e36673051929268ab86d422c00R58

Reply:
no, your changed dropped line 58. I'd like to keep that marker.